foreman/data_refinery_foreman/surveyor/test_transcriptome_index.py

```diff
@@ -51,6 +51,9 @@ def test_correct_index_location(self):
         surveyor = TranscriptomeIndexSurveyor(survey_job)
         files = surveyor.discover_species()[0]
 
+        # Make sure the organism object got created.
+        Organism.objects.filter(name="DANIO_RERIO")
```

Comment:
is there an assertion missing here?

Reply:
Nope! I wanna make sure we don't get an exception, and there's no assertion for that because it's kinda asserted by default. I can leave a comment to make that more explicit though.

Reply:
hmm but `Organism.objects.filter` should never throw an exception, or I'm missing something?

Reply:
Good call, I meant to use `get()` instead of filter, fixed now.